packages/front-end/src/app/Utils/backend-service.ts

```diff
@@ -0,0 +1,3 @@
+export default function backendUrl(): string {
+  return 'http://localhost:3000';
+}

```

Comment:
nit: use const instead of function
```suggestion
export default 'http://localhost:3000';
```